packages/dd-trace/src/service-naming/schemas/util.js

```diff
@@ -2,4 +2,22 @@ function identityService ({ tracerService }) {
   return tracerService
 }
 
-module.exports = { identityService }
+function getFormattedHostString ({ host, port }) {
```

Comment:
It it intentional that `getFormattedHostString ({ port:123 })` returns `'123'`?

Reply:
Eh, probably a non-issue

Reply:
We're relying on sessionDetails always having a host in http / http2, which sounds like a fair assumption to me.